server/app/mutations/grids/update.rb

```diff
@@ -23,6 +23,14 @@ class Update < Mutations::Command
       array :default_affinity do
         string
       end
+      hash :logs do
+        required do
+          string :forwarder, matches: /^(fluentd|none)$/ # Only fluentd now supported, none removes log shipping
```

Comment:
Should I change this to be "manual" validation in the mutation to make it give better errors?